vllm/attention/layer.py

```diff
@@ -23,6 +23,18 @@
 from vllm.platforms import _Backend, current_platform
 from vllm.utils import direct_register_custom_op
 
+try:
+    from xformers import ops as xops
+
+    if current_platform.is_cuda() and current_platform.has_device_capability(
+            100):
+        # Xformers FA is not compatible with B200
+        USE_XFORMERS_OPS = False
+    else:
+        USE_XFORMERS_OPS = True
+except ImportError:
+    USE_XFORMERS_OPS = False
```

Comment:
This import should not be tried in such a global place. Please make it lazy somehow